common/src/api/external/http_pagination.rs

```diff
@@ -45,7 +45,7 @@ use crate::api::external::Name;
 use crate::api::external::ObjectIdentity;
 use crate::api::external::PaginationOrder;
 use dropshot::HttpError;
-use dropshot::PaginationParams;
+pub use dropshot::PaginationParams;
```

Comment:
This is necessary due to the metrics params using `PaginationParams` directly. See that addition here: https://github.com/oxidecomputer/omicron/pull/2008/files#diff-aee01ef600c8eb547421397e5d89c41a67dd6e6ca6033676cac47b7909e1e344R123-R130

Reply:
Not necessary anymore?